packages/node/src/integrations/tracing/nest/types.ts

```diff
@@ -8,6 +8,7 @@ export interface MinimalNestJsExecutionContext {
     // according to official types, all properties are required but
     // let's play it safe and assume they're optional
     getRequest: () => {
+      routeOptions?: any;
```

Comment:
instead of `any`, can we either use `unknown` or some more specific type here? This also ensures we guard this so nothing can blow up.

Reply:
should we also type this object so that it's either `routeOptions` or `route`? (fastify vs. express).

This makes the `if else` a little more safe to write.